eng/pipelines/pipeline-generation.yml

```diff
@@ -16,40 +16,40 @@ jobs:
         Branch: master
         Prefix: java
         PublicOptions: ''
-        InternalOptions: '--variablegroup 1 --variablegroup 58 --variablegroup 76 --variablegroup 56'
+        InternalOptions: '--variablegroup 1 --variablegroup 58 --variablegroup 76 --variablegroup 56 --variablegroup 93'
```

Comment:
Supports APIView?

Reply:
Yes. New variable group to support APIView tool